LibraBFT/Impl/Properties/VotesOnce.agda

```diff
@@ -207,7 +207,7 @@ module LibraBFT.Impl.Properties.VotesOnce where
   vo₁ : VO.ImplObligation₁
   -- Initialization doesn't send any messages at all so far.  In future it may send messages, but
   -- probably not containing Votes?
-  vo₁ r (step-init _) _ _ m∈outs _ _ _ _ _ _ _ _ = ⊥-elim (¬Any[] m∈outs)
```

Comment:
I guess you removed some `_`s when experimenting with a different version of the obligation, and didn't put them all back.  It doesn't matter.  In fact, you could omit all of them after `m∈outs`.

Reply:
You are right again! I omitted all of them after `m∈outs` now.